S23/vivekkatari/Main.java

```diff
@@ -0,0 +1,45 @@
+import java.util.Locale;
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.print("Enter a word: ");
+        String word = scanner.nextLine();
+        System.out.print("Enter the language code (e.g. en-US, tr-TR, el-GR): ");
+        String languageCode = scanner.nextLine();
+        String lowerCaseWord = LowerCase.toLowerCase(word, languageCode);
+        System.out.println("Lowercase word: " + lowerCaseWord);
+        scanner.close();
+    }
+
+    public static class LowerCase {
+        public static String toLowerCase(String word, String language) {
+            Locale locale = language != null && !language.isEmpty() ? Locale.forLanguageTag(language) : Locale.getDefault();
+
+            String lowerCaseWord;
+            String languageTag = locale.toLanguageTag();
+
+            if (languageTag.startsWith("tr") || languageTag.startsWith("az")) lowerCaseWord = word.replace("I", "ı");
```

Comment:
I would be nice if you use brackets .

Reply:
Thanks for the suggestion The functionality is work as expected so there would be no difference in updating the code.